test/parallel_api/algorithm/alg.modifying.operations/shift_left_right.pass.cpp

```diff
@@ -48,6 +53,33 @@ struct test_shift
     }
 
 #if TEST_DPCPP_BACKEND_PRESENT
```

Comment:
Do we still need this check? seems like we just put it upper

Reply:
Upper check was closed before.